pkg/user/user.go

```diff
@@ -1,32 +1,49 @@
 package user
 
 import (
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/go-pg/pg/v9"
+
 	"github.com/chrsep/vor/pkg/auth"
-	"github.com/chrsep/vor/pkg/postgres"
 	"github.com/chrsep/vor/pkg/rest"
-	"github.com/go-chi/chi"
-	"net/http"
+	"github.com/chrsep/vor/pkg/user/store"
 )
 
-type Store interface {
-	GetUser(userId string) (*postgres.User, error)
-	GetSchools(userId string) ([]postgres.School, error)
-}
+var (
+	userStore store.UserStore
+)
 
-type server struct {
-	rest.Server
-	store Store
+type (
+	UserHandler interface {
+		GetUser() rest.Handler
+		GetSchools() rest.Handler
+	}
```

Comment:
I know that that it might be useful for mocking to have the server struct and UserHandler interface. But what are the example usages?

Reply:
iya, ini sbnrnya gw gak mau buat interface cuma gw lg bingung gmna wkwkwk. bntr gw coba ubah lg ntar